.editorConfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
Should we add this type of files to .gitignore?

Reply:
Nvm, I just removed it, if we find bad line endings in the future we should check how to fix it with lint